libraries/hatch-server-user-management/src/UserManagementError.ts

```diff
@@ -0,0 +1,17 @@
+export enum UserManagementErrorCodes {
```

Comment:
Since this will be used by the client SDK -- both to interpret responses and to re-throw to the client, we'll probably want to move it to `hatch-user-management-client`

Reply:
Unless there's a good reason not to, the client SDK should probably just implement the existing UserServiceClient interface (which should also move to the client SDK library), since UserManagementController's APIs map to that 1:1.

Reply:
updated